lib/galaxy/util/permutations.py

```diff
@@ -83,7 +83,8 @@ def __extend_with_matched_combos(single_inputs, multi_inputs):
         if multi_input_key == first_multi_input_key:
             continue
         if len(multi_input_values) != len(first_multi_value):
-            raise InputMatchedException()
+            raise InputMatchedException("Received %d inputs for '%s' and %d inputs for '%s, these should be of equal length" %
```

Comment:
There's a missing `'` after the last `%s`.

Reply:
rebased that with the fix, thanks @nsoranzo